Documentation/markdown/GitProcess.md

```diff
@@ -0,0 +1,133 @@
+# Git Workflow
```

Comment:
Suggestion for the structure of the document: Maybe separate the GitProcess.md into two parts: 
(1) GitSetup (which is once-off): Prerequisites, Fork
(2) GitWorkflow (which is to explain how to work with Git all the time):
- Updating from remote
-  Creating a branch (move this here from first part that is currently under "Submitting changes" and include branch naming conventions here
- Submit changes
- Pull request

(In this order as it is then the same as the actual workflow and especially so that you have latest version of tumcms development OIP before starting to make any local changes.)

Reply:
Done